CLI: add install `--from-download` for sssp and pseudo-dojo (#135)
aiidateam/aiida-pseudo: docs/source/troubleshooting.rst

Troubleshooting
###############

.. _troubleshooting:automated-fail:

The automated install commands fail
===================================

These failures are often due to unstable internet connections causing the download of the pseudopotential archive from the web to fail.

> In this case, it is possible to download an ``.aiida_pseudo`` archive of the established family, transfer them to the machine where the pseudopotentials need to be installed and install from a directory.

Reviewer: ```suggestion
In this case, it is possible to download an ``.aiida_pseudo`` archive of the established family, transfer them to the machine where the pseudopotentials need to be installed and install directly from the archive.
```

To download the archive, use the ``--download-only`` option:

.. code-block:: console

    $ aiida-pseudo install sssp --download-only
    Report: downloading patch versions information...  [OK]
    Report: downloading selected pseudopotentials archive...  [OK]
    Report: downloading selected pseudopotentials metadata...  [OK]
    Success: Pseudopotential archive written to: SSSP_1.1_PBE_efficiency.aiida_pseudo

> You can then transfer the downloaded archive to the machine where it should be installed use the ``--from-download`` option of the corresponding install command:

Reviewer: ```suggestion
The downloaded archive can be installed through the ``--from-download`` option of the corresponding install command:
```
